### ISSUE TITLE
Sometimes can't track specific spell's cooldown after changing talent loadout in instance (Evoker only?)

### ISSUE THREAD

**Issue by Elnarfim** [WeakAuras/WeakAuras2#5551]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues.

### Description

I'm not sure that is happened to every classes. I've experienced this matter with preservation evoker only because I'm a healer user
For me, stasis field WA was broken. It couldn't track cooldown unitl changing an option to Exact Spell Match
UI reload can't solve it and stasis field is learned in two loadouts (before and after)
I was in mythic+ at that time and the WA was still broken after mythic+ (changing talent loadout was happened before start)
It was unable in the city and worked again after relogin only (If you don't choose Exact Spell Match option to track)

In conclusion, there are two solutions for the bug
1. Changing option to Exact Spell Match
2. ReLogin

### WeakAuras Version

5.18.0

### World of Warcraft Flavor

Retail (Default)

### World of Warcraft Region

TW/Asia

### Tested with only WeakAuras

I got this issue with only WeakAuras enabled

### Reproduction Steps

1. make some WAs to track spells cooldown especially stasis field (it was broken for me)
2. You should have 2 talent loadouts at least and be in instance (raid or mythic+ anywheres)
3. Change talent loadout to different one in instance

**Comment by InfusOnWoW** [comment]
Duplicate of https://github.com/WeakAuras/WeakAuras2/issues/5550

It's a Blizzard bug.
